custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -406,10 +406,19 @@ def _get_person_case(self, episode_case):
 
     @staticmethod
     def _get_husband_father_name(person_case_properties):
-        husband_father_name = person_case_properties.get('husband_father_name')
+        husband_father_name = sanitize_text_for_xml(person_case_properties.get('husband_father_name', ''))
+        # check after sanitize for xml to consider cases where it would become blank after being sanitized
         if husband_father_name:
             return husband_father_name
-        return person_case_properties.get('last_name', '')
+        return sanitize_text_for_xml(person_case_properties.get('last_name', ''))
+
+    @staticmethod
+    def _get_address(person_case_properties):
+        # API for Address with char sanitize_text_for_xml ',' returns Invalid data format error
```

Comment:
just that the API where we are sending this gives invalid data format error in case this has ',' within in even after sanitising for xml so we need to remove that separately @calellowitz